_docs/getting-started/design.md

```diff
@@ -11,7 +11,7 @@ last-modified: 2017-10-24
 
 
 
-<section>
+<section class="static-section">
```

Comment:
These ones are the only ones without it, it looks like, so I'd add them in here.